editors/code/syntaxes/clarity.tmLanguage.json

```diff
@@ -153,7 +153,7 @@
     },
     "tuple-key": {
       "name": "entity.name.type.clarity",
-      "match": "(?x)\n  ([a-z][a-zA-Z0-9_-]*)(?=:)\n"
+      "match": "(?x)\n  ([a-z][a-zA-Z0-9_-]+)(?=:)\n"
```

Comment:
Doesn't this change exclude single letter key names? Also, is there any reason that the key name can't start with a capital letter or an '_'?

Reply:
Good catch, for some reason I had an outdated version of that file on my branch.

Reply:
When you reverted, there is still a change to this file, just the newline at the end. It would be cleaner if there were no changes to this file.